ConfigSpace/forbidden.py

```diff
@@ -89,7 +92,7 @@ def __init__(self, hyperparameter, value):
                              "'%s'" % (hyperparameter, str(value)))
         self.value = value
 
-    def is_forbidden(self, instantiated_hyperparameters, strict=True):
+    def is_forbidden(self, instantiated_hyperparameters: Hyperparameter, strict: bool=True) -> bool:
```

Comment:
changed the type of instantiated_hyperparameters to Dict[str, Union[None, str, float, int]] .
all versions of _is_forbidden are annotated except abstract methods